it/it-plugins/batch-plugin/src/main/java/com/sonarsource/BatchPlugin.java

```diff
@@ -40,7 +41,7 @@ public List getExtensions() {
       DumpSettingsInitializer.class,
       RaiseMessageException.class,
       TempFolderExtension.class,
-      WaitingSensor.class);
+      WaitingSensor.class
+      ));
```

Comment:
This trailing parenthesis difference between Eclipse and IntelliJ formatters is very annoying :)